src/main/java/org/example/userproject1/service/PhoneServise.java

```diff
@@ -0,0 +1,60 @@
+package org.example.userproject1.service;
+
+import jakarta.persistence.EntityNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.example.userproject1.entity.Phone;
+import org.example.userproject1.entity.User;
+import org.example.userproject1.repository.PhoneRepository;
+import org.example.userproject1.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@Service
+public class PhoneServise {
+    private final PhoneRepository phoneRepository;
+    private final UserRepository userRepository;
+    public List<Phone> userContacts(long id) {
+        User user;
+        try {
+            user = userRepository.findById(id).orElseThrow(EntityNotFoundException::new);
```

Comment:
Here you throw exception, catch it and rethrow another one

Instead, what you should do is create custom exception UserNotFoundException and throw it directly

User user = userRepository.findById(id)
            .orElseThrow(() -> new UserNotFoundException("User with id=" + id + " does not exist"));